auth/authController.js

```diff
@@ -76,20 +76,16 @@ const login = async (req, res, next) => {
   }
 };
 const logout = async (req, res, next) => {
-  const token = req.user.token;
-  if (!token) {
+  if (!req.user.token) {
```

Comment:
To też w sumie nie potrzebne bo auth  pierwszy wyrzuci 401 więc tu jest to zduplikowane